### DISCUSSION TITLE
3D Pie

### DISCUSSION THREAD

**Discussion by Warren-Whitfield** [pelicanmapping/osgearth#2527] [Support Q&A] [answered]
Is it possible to draw a 3D Pie using something like a CircleNode?
For example, something like this, but that extends in altitude as well?

![image](https://github.com/gwaldron/osgearth/assets/159110675/072131df-5199-44ac-9106-0ee6acdf7d86)

**Comment by gwaldron** [comment, marked as answer]
No, you will have to roll your own. Good luck :)

**Comment by Warren-Whitfield** [comment]
Cool, thx!

**Reply by kahlenberg** [reply]
It might be possible with `osgEarth::Symbology::GeometryFactory::createArc()`
